che-incubator/che-code
# chore: remove redundant github action

### What does this PR do?
Removes **image-publish-stable** github action as it is triggered only when pushing to obsollette '1.*.x' branch

### What issues does this PR fix?

No issue

### How to test this PR?
Is not needed

To see when the last image has been built and uploaded with this action:
- go to https://quay.io/repository/che-incubator/che-code?tab=tags
- filter tags by `1.`

The last `1.70.x-20537ee` tag was pushed around 10 months ago
